### ISSUE TITLE
Staging Review finding: Update styling on color only alert for past events

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#97240]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** CMS Team
**Product:** Next Build and Next Build's Events and Event Listing Templates
**Feature:** Events and Event Listing Templates

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @allison0034 (Design), @briandeconinck (Accessibility)

### Description

If an event has already happened there there is red text on the page “This event already happened.” There is no other formatting around the message. This is both an accessibility issue and different from what we have in the design system.

<details><summary>Link, screenshot or steps to recreate</summary>

![Screen shot](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/96324-Allison.png?raw=true)

</details>

## Recommended action

Replace the color only alert with the [standard alert style](https://design.va.gov/components/alert/#error-alert). 

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Alert ](https://design.va.gov/components/alert)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jilladams** [comment]
This is a difference from production, where the red text does not appear on past events. FYI @timcosgrove @gracekretschmer-metrostar . Could we just remove this for now, until Sitewide can help think about a design treatment for distinguishing past and future events? 

cc @FranECross @thejordanwood

**Comment by jilladams** [comment]
I no longer see this past red text on past events, in production (AP launched last night, so prod events are now Next): https://www.va.gov/wichita-health-care/events/74545/

Might be possible to close this: 
![Image](https://github.com/user-attachments/assets/b2152ae2-34a1-4f3e-adec-2ef58bc27e68)
